en/docs/using-wso2-identity-server/access-control-for-developers.md

```diff
@@ -4,4 +4,4 @@ The following topics list out key access control concepts relevant for
 Developers.
 
 -   [Writing a Custom Policy Info
-    Point](_Writing_a_Custom_Policy_Info_Point_)
+    Point](../../using-wso2-identity-server/writing-a-custom-policy-info-point)
```

Comment:
this link has been reverted back to the wrong link format

Reply:
The link format is correct.